pycheribuild/projects/run_qemu.py

```diff
@@ -102,8 +104,11 @@ def process(self):
             self.dependencyError("Kernel is missing:", self.currentKernel,
                                  installInstructions="Run `cheribuild.py cheribsd` or `cheribuild.py run -d`.")
         if self.diskImage:
-            if len(self._diskOptions) == 0:
-                self._diskOptions = ["-hda", self.diskImage]
+            if self.virtioDisk:
+                diskOptions = ["-drive", "if=none,file=" + str(self.diskImage) + ",id=drv,format=raw",
+                               "-device", "virtio-blk-device,drive=drv"]
+            else:
+                diskOptions = ["-hda", self.diskImage]
```

Comment:
Should we be using virtio disks for MIPS as well?

Reply:
Maybe? Someone should do some benchmarking :)